package.json

```diff
@@ -13,7 +13,7 @@
   "dependencies": {
     "linguist-languages": "^7.5.1",
     "mem": "^8.0.0",
-    "php-parser": "https://github.com/moddengine/php-parser#c71ee4b5c7bab44dbaf5cb9e9bce7e69b45c23b8"
+    "php-parser": "hhttps://github.com/glayzzle/php-parser#cca83041d938b972c7ef74648f29de03777b0ea4"
```

Comment:
@czosel here bug

Reply:
That's not the latest version you're looking at. It currently links to the right commit on `master`, and the latest version of this PR as well.

The parses doesn't understand the extra `,`.